### DISCUSSION TITLE
How to get orgazation mail id ?

### DISCUSSION THREAD

**Discussion by sakthivelj** [community/community#79291] [New to GitHub]
### Select Topic Area

Question

### Body

Hello, We have created an organization for our company, but we forgot which email ID we used to create it. How can we find out the email address associated with our organization at Croatia-technologies https://github.com/coratia-technologies?

I have raised a ticket, but I haven't received any response yet https://support.github.com/ticket/personal/0/2481497. Could you please let me know how long it will take to get a response?

Thanks
Sakthivelj

**Comment by TalhaT298** [comment]
Check the organization's profile on GitHub. Some organizations may provide contact information in their profile description or the README file in their repositories. Look for sections where they might have listed email addresses or contact details.

**Reply by sakthivelj** [reply]
@TalhaT298 This is ours only, nothing is available that's why I posted in discussion?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sakthivelj ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by MPS-SteveGornick** [comment]
In the same boat, myself.    Support ticket went a week with no reply.  I'm on the Github Free for Organizations plan, and see that it only includes Community Support.  Oh well.

I suppose a way to escalate would be to upgrade to a different [Github Plan](https://github.com/organizations/plan) (Github Teams, or Enterprise?) and then use the Web support that either of those plans includes.

**Comment by GunaPalanivel** [comment]
Finding the email address:

Unfortunately, there isn't a direct way to view the email address linked to your organization publicly on GitHub. However, here are some options you can try:

Check your organization settings: If you have access to the organization settings, you might find the email address listed under the "Billing and subscriptions" section.
Review communication from GitHub: Look through any emails you might have received from GitHub related to the organization creation or other activity. These emails might include the associated email address.
Contact GitHub support: If you're still unable to find the email address, you can contact GitHub support directly. They might be able to access and share the information with you after verifying your identity.
Support ticket response:

The response time for support tickets can vary depending on the complexity of the issue and the current workload of the GitHub support team. Unfortunately, I cannot provide a specific timeline for your ticket (0/2481497). However, you can expect a response within the next few business days.

Additionally:

You can track the status of your ticket by logging into your GitHub account and navigating to the "Support" section.
If you haven't received a response within a reasonable timeframe, you can try sending a follow-up message to GitHub support politely inquiring about the status of your ticket.

I hope this information helps!
